tests/unit/lms/resources/_js_config_test.py

```diff
@@ -190,6 +291,15 @@ def no_lti_user(pyramid_request):
     pyramid_request.lti_user = None
 
 
+@pytest.fixture
+def pyramid_request(pyramid_request):
+    pyramid_request.params["lis_result_sourcedid"] = "example_lis_result_sourcedid"
+    pyramid_request.params[
+        "lis_outcome_service_url"
+    ] = "example_lis_outcome_service_url"
+    return pyramid_request
```

Comment:
It could be done but the cure might be worse than the disease.

We should probably just have a standard LTI launch request fixture in `conftest.py` with all the standard params common to the most common type of launch (a student launching an assignment) across all LMS's. That way standard params like `lis_result_sourcedid` would be in the request all the time, so tests wouldn't have to keep adding them, but at the same time they wouldn't be in the test code (only in `conftest.py`) so tests that don't need them wouldn't look confusing. You'd then have more specific request fixtures for things like a Canvas request (includes custom Canvas params) or a teacher request (`lis_result_sourcedid` is missing) etc, and those could either be local or global as needed.

That's a fair bit of test fixture refactoring though, so something for later